edit/harvesterhci.io.setting/additional-ca.vue

```diff
@@ -0,0 +1,40 @@
+<script>
+import CreateEditView from '@/mixins/create-edit-view';
+import FileSelector, { createOnSelected } from '@/components/form/FileSelector';
+import TextAreaAutoGrow from '@/components/form/TextAreaAutoGrow';
+
+export default {
+  name: 'HarvesterAdditionalCA',
+
+  components: { FileSelector, TextAreaAutoGrow },
+
+  mixins: [CreateEditView],
+
+  data() {
+    return { errors: [] };
+  },
+
+  methods: { onKeySelected: createOnSelected('value.value') },
+};
+</script>
+
+<template>
+  <div>
+    <div class="row mb-20">
+      <div class="col span-12">
+        <FileSelector
+          class="btn btn-sm bg-primary mb-10"
+          :label="t('generic.readFromFile')"
+          @selected="onKeySelected"
+        />
```

Comment:
I think should add `accept` to filter the ca file.

`accept="application/x-x509-ca-cert,text/plain,.pem,.crt"`

![image](https://user-images.githubusercontent.com/18737885/144019711-d34fb375-da10-4371-b3e3-8fdb7c52204d.png)

Also, the placeholder can be `Paste in the certificate and any necessary chain certificates, starting with -----BEGIN CERTIFICATE-----`